Running cluster for deep fits
180013 180014 normal
Running object 180013
WARNING: Could not memory map array with mode='readonly', falling back to mode='denywrite', which means that the array will be read-only [astropy.io.fits.file]
Min gap is: 0.0
New time_gap is :0.004
Number of observations now:  85
Min gap is: 0.00390625
New time_gap is :0.00790625
Number of observations now:  137
Min gap is: 0.00390625
New time_gap is :0.00790625
Number of observations now:  159
Min gap is: 0.00390625
New time_gap is :0.00790625
Number of observations now:  188
Counts: {'g': 85, 'r': 136, 'i': 159, 'z': 188}
Mu: {'g': 0.21919668, 'r': 0.38018817, 'i': 0.4685004, 'z': 0.5376363}
Variable??
0.019397891076314153
1.2879440733729368
Non-Variable source in g
Variable??
0.019301385157176408
2.2214127575545137
Non-Variable source in r
Variable??
0.024607583864278013
1.7037104807639982
Non-Variable source in i
Variable??
0.03728937847391211
1.3484682876528054
Non-Variable source in z
Variable:    0/4
Var Bands:   
Slope:       [0.5315367027352689, 1, 0.4783091641342405, 0.3147406399430821]
Slope Error: [0.19883695426012016, 0, 0.10332798874888406, 0.1127061627554737]
intercepts:  [0.009546428819055541, 0, 0.0263087201342318, 0.07961612163407361]
Int Error:   [0.02745810908865201, 0, 0.013464207695098365, 0.018340678153864576]
StDev:       [0.040024832, 0.057724766, 0.07817671, 0.10655316]
Not variable, closing plots & going to next row
Successfully finished!
Rows 180013 to 180014 ran in 3.0211 seconds
